### PR TITLE
Added interface for crymol program

### PR DESCRIPTION
**Pull request by dr-geeno**
crymol is a program part of the GNXAS package for fitting EXAFS data (for more information, see [here](http://gnxas.unicam.it/pag_gnxas.html)). I am a contributor to the code.

As far as I know, there was no easy way to convert from a cif file to its input, so instead of creating one from zero, I added it to cif2cell, which I found by looking around for codes that converts cif files. I tested on the files inside cifs folder and it runs on all of them without problems (alloys are not supported, but using --force-alloy created a reasonable input file that can be easily modified). For simplicity, it just uses the conventional unit cell, in principle the use of the primitive unit cell is also possible, but require to insert also the symmetry operations in the input file and since for the purpose of the program the two are equivalent I didn't bother. 

I also removed all the trailing spaces in binaries/cif2cell because I have slight OCD :)

I would be happy if you consider adding it to the official repository. Thank you!

### CODE REVIEW

File: cif2cell/ESPInterfaces.py [hunk 1]
```diff
@@ -3361,3 +3362,68 @@ def __str__(self):
...
             filestring += "Tv  "+str(l)+"\n"
         return filestring
+
+################################################################################################
+# GNXAS CRYMOL FILE
+
+class CRYMOLFile(GeometryOutputFile):
+    """
+    Class for storing the geometrical data needed for outputting an input file
+    for crymol program of the GNXAS package and the method __str__ that outputs
```
Comment (ltalirz) [inline cif2cell/ESPInterfaces.py:3372]:
```suggestion
    for the crymol program of the GNXAS package and the method __str__ that outputs
```

File: cif2cell/ESPInterfaces.py [hunk 2]
```diff
@@ -3361,3 +3362,68 @@ def __str__(self):
...
+                  self.crystal_system()+" crystal systems in crymol.")
+
+        filestring += "%i\n" % (self.cell.natoms())
+        index = 1
+        width = 2
+        for a in self.cell.atomdata:
+            for b in a:
+                filestring += str(index).rjust(width)+"  '"+b.spcstring().rjust(width)\
+                             +"'  "+str(b.position)+" 12 0.0 0.0\n"
+                index += 1
```
Comment (ltalirz) [inline cif2cell/ESPInterfaces.py:3412]: perhaps add somewhere a "!start section to be modified" comment (not sure whether that's allowed by the format)

File: cif2cell/ESPInterfaces.py [hunk 3]
```diff
@@ -3361,3 +3362,68 @@ def __str__(self):
...
+        filestring += "1,1\n"
+        filestring += "0.10\n"
+        filestring += "1\n"
+        filestring += "1\n"
+        filestring += "CIP,0. [insert Core ionization energy in Ry!]\n"
+        filestring += "3,-0.2,120.0,0.04\n"
+        filestring += "5,-1\n"
+        filestring += "'S',1,50.,15.,'S',' ','S'\n\n"
+        filestring += self.docstring
+        filestring += "!cif2cell only convert the geometrical structure, remember to change the other parameters!"
```
Comment (ltalirz) [inline cif2cell/ESPInterfaces.py]: Just mentioning that there are different possible approaches here - you could (a) take the parameters for a specific system (might be worth mentioning in the file) (b) remove all values for which a sensible default cannot be provided and only put suggestions in comments, ...

You will know best, I leave this up to you

File: cif2cell/ESPInterfaces.py (hunk 2 above)
Comment (dr-geeno) [reply cif2cell/ESPInterfaces.py:3412]: In this specific part is not required, the numbers at the end of the line are not used most of the time, but still required by the program.

File: cif2cell/ESPInterfaces.py (hunk 3 above)
Comment (dr-geeno) [reply cif2cell/ESPInterfaces.py]: I wanted to do approach (b), but I over-did it with the default values. Made another commit, only left values where the default is good.

**Review by dr-geeno — COMMENTED** [review]
Removed parameters for which a standard default would be generally not good

**Pull request merged**